gusto/common_forms.py

```diff
@@ -194,3 +198,67 @@ def diffusion_form(test, q, kappa):
     form = -inner(test, div(kappa*grad(q)))*dx
 
     return diffusion(form)
+
+
+def split_continuity_form(equation):
+    u"""
+    Loops through terms in a given equation, and splits continuity terms into
```

Comment:
Could you mention that this will split up *all* conservative terms in the equation?

Reply:
Done! Thanks Tom